packages/kit/test/apps/basics/test/test.js

```diff
@@ -197,7 +197,23 @@ test.describe('Scrolling', () => {
 	}) => {
 		await page.goto('/anchor');
 		await clicknav('#last-anchor-2');
-		expect(await page.evaluate(() => pageYOffset === 0)).toBeTruthy;
+		expect(await page.evaluate(() => scrollY === 0)).toBeTruthy;
+	});
+
+	test('scroll is restored after hitting the back button', async ({ back, clicknav, page }) => {
+		await page.goto('/anchor');
+		await page.click('#scroll-anchor');
+		const originalScrollY = /** @type {number} */ (await page.evaluate(() => scrollY));
+		await clicknav('#routing-page');
+		await back();
+		expect(page.url().substring(page.url().indexOf('/', 8))).toEqual('/anchor#last-anchor-2');
```

Comment:
This is a bit hard to parse, could this be refactored as:
```suggestion
		expect(page.url()).toBe(baseURL + '/anchor#last-anchor-2');
```

The `baseURL` needs to be destructured from the test function.

Reply:
thanks. done